samples/counter/ios-shared/src/commonMain/kotlin/com/example/redwood/counter/ios/IosText.kt

```diff
@@ -32,5 +32,9 @@ class IosText : Text<UIView> {
 
   override fun text(text: String?) {
     value.text = text
+
+    // This very simple integration wraps the size of whatever text is entered. Calling
+    // this function will update the bounds and trigger relayout in the parent.
+    value.sizeToFit()
```

Comment:
Can do. I don't think it's the perfect solution for general usage, but it's certainly enough to achieve correctness at the cost of possibly doing two text layouts (since the parent may constrain the view and cause wrapping).